modules/ensemble/lib/widget/confirmation_input.dart

```diff
@@ -201,7 +203,7 @@ class ConfirmationInputState extends framework.WidgetState<ConfirmationInput>
       keyboardType: widget.keyboardType,
       otpPinFieldDecoration: controller.fieldType?.otpPinField ??
           OtpPinFieldDecoration.defaultPinBoxDecoration,
-      otpPinFieldInputType: controller.obscureText?.otpPinType ?? OtpPinFieldInputType.none,
+      otpPinFieldInputType: controller.obscureText == true ? OtpPinFieldInputType.password : OtpPinFieldInputType.none,
```

Comment:
why is this needed?

Reply:
we are using `otp_pin_field` module to create `confirmationInput` and obsure property is defined here https://github.com/EnsembleUI/ensemble/blob/bf9a9948913d4c7c610399c72238a6076b007406/modules/otp_pin_field/lib/src/otp_pin_field_widget.dart#L17 with dataType being enum defined at https://github.com/EnsembleUI/ensemble/blob/bf9a9948913d4c7c610399c72238a6076b007406/modules/otp_pin_field/lib/src/otp_pin_field_input_type.dart#L1. and its manipulation that how it will be displayed is done in `otp_pin_field_state.dart` that's why adjusted the `confirmationInput` code accordingly.